src/coreComponents/common/Format.hpp

```diff
@@ -22,7 +22,9 @@
 #endif
 
 #ifdef GEOSX_USE_FMT
+#ifndef FMT_HEADER_ONLY
```

Comment:
Is this defined somewhere else?? Umpire??

Reply:
I am guessing somewhere in the new `RAJA` suite release.